src/lang/en-US.ts

```diff
@@ -307,7 +316,7 @@ const ConnectorsPage: ResolvedIntlConfig['messages'] = {
     'connectorTable.data.external_url': `Homepage`,
     'connectorTable.data.actions': `Actions`,
     'connectorTable.actionsCta.capture': `Capture`,
-    'connectorTable.actionsCta.materialization': `Materialization`,
+    'connectorTable.actionsCta.materialization': `Materialize`,
     'connectors.header': `Connectors`,
     'connectors.main.message1': `There are no connectors available matching your search.`,
     'connectors.main.message2': `If you'd like to request a connector for a system that isn't yet supported, {docLink}.`,
```

Comment:
Assuming this is what gets shown when a user searches for a connector and there's no results, it might be best to have this link directly to the [page for creating a new GH issue](https://github.com/estuary/connectors/issues/new?assignees=&labels=new+connector&template=request-new-connector-form.yaml&title=Request+a+connector+to+%5Bcapture+from+%7C+materialize+to%5D+%5Byour+favorite+system%5D)

Reply:
That would be an easy swap to make at this time, thank you for the suggestion. Will update.